junebug/channel.py

```diff
@@ -259,6 +259,7 @@ def _application_config(self):
             'redis_manager': self.config.redis,
             'inbound_ttl': self.config.inbound_message_ttl,
             'outbound_ttl': self.config.outbound_message_ttl,
+            'message_rate_bucket': self.config.inbound_message_rate_bucket,
```

Comment:
I can't think of any reason to have it separate, so I think I'll change it to `metric_window`